qiskit/transpiler/passes/scheduling/padding/base_padding.py

```diff
@@ -130,16 +149,19 @@ def run(self, dag: DAGCircuit):
         # Add delays until the end of circuit.
         for bit in new_dag.qubits:
             if circuit_duration - idle_after[bit] > 0:
-                node = new_dag.output_map[bit]
-                prev_node = next(new_dag.predecessors(node))
-                self._pad(
-                    dag=new_dag,
-                    qubit=bit,
-                    t_start=idle_after[bit],
-                    t_end=circuit_duration,
-                    next_node=node,
-                    prev_node=prev_node,
-                )
+                if self.target is None or self.target.instruction_supported(
+                    "delay", qargs=(bit_indices[bit],)
+                ):
+                    node = new_dag.output_map[bit]
+                    prev_node = next(new_dag.predecessors(node))
+                    self._pad(
+                        dag=new_dag,
+                        qubit=bit,
+                        t_start=idle_after[bit],
+                        t_end=circuit_duration,
+                        next_node=node,
+                        prev_node=prev_node,
+                    )
```

Comment:
Do you think we should log a message to indicate to users why we're not putting a delay on the the qubit (maybe DEBUG level)? I'm not sure it's needed but I'm wondering if it'll be confusing for users of the scheduling passes if they ask for scheduling and there are qubits without delays?